src/MultiDBConfigDialog.cpp

```diff
@@ -12,6 +12,8 @@
 #include "DBInterface.hpp"
 #include "XRockIOLibrary.hpp"
 #include <configmaps/ConfigMap.hpp>
+#include "BundleSelectionDialog.hpp"
```

Comment:
How does this fit into this pull request?

Reply:
Is this file committed?

Reply:
sorry. I created my new branch on the head of another branch . now I committed both: with the bundle section dialog it will allow user to select the bundle from the dialog incase he didnot do it initially

Reply:
And what happens if there is no bundle to select from?